gef-ui/server/controllers/confirm-cover-start-date/index.js

```diff
@@ -16,17 +16,21 @@ const setError = (field, message) => validationErrorHandler({
   errMsg: message,
 });
 
-const updateCoverStartDate = async (facilityId, { coverStartDate, shouldCoverStartOnSubmission }, dealId, editorId) => {
+const updateCoverStartDate = async (facilityId, { coverStartDate, shouldCoverStartOnSubmission }, dealId, editorId, userToken) => {
```

Comment:
Lots of positional arguments - might be better to pass an object and destructure?